src/main/kotlin/org/malachite/estella/people/api/HrPartnerController.kt

```diff
@@ -30,15 +27,15 @@ class HrPartnerController(
     @CrossOrigin
     @GetMapping
     fun getHrPartners(): ResponseEntity<MutableIterable<HrPartner>> {
-        return ResponseEntity(hrPartnerService.getHrPartners(), HttpStatus.OK)
+        return ResponseEntity.ok(hrPartnerService.getHrPartners())
     }
 
     @CrossOrigin
     @GetMapping("/{hrPartnerId}")
     fun getHrPartner(@PathVariable("hrPartnerId") hrPartnerId: Int): ResponseEntity<HrPartner> {
         val partner: HrPartner = hrPartnerService.getHrPartner(hrPartnerId)
 
-        return ResponseEntity(partner, HttpStatus.OK)
+        return ResponseEntity.ok(partner)
```

Comment:
Ogólnie chyba pora sobie wybrać jeden sposób robienia ResponseEntity, bo co refaktor, to każdy refaktoruje po swojemu XD

Reply:
Tutaj chyba radek wszędzie pozamieniał na te buildery (imo dobrze, bo to ładniejsze)